[2025-03-09T17:10:04.308+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-09T17:10:04.332+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_fire_incidents_dag.load_data manual__2025-03-09T17:09:50.292043+00:00 [queued]>
[2025-03-09T17:10:04.340+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_fire_incidents_dag.load_data manual__2025-03-09T17:09:50.292043+00:00 [queued]>
[2025-03-09T17:10:04.340+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-09T17:10:04.355+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_data> on 2025-03-09 17:09:50.292043+00:00
[2025-03-09T17:10:04.365+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=18954) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-09T17:10:04.367+0000] {standard_task_runner.py:72} INFO - Started process 18956 to run task
[2025-03-09T17:10:04.367+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_fire_incidents_dag', 'load_data', 'manual__2025-03-09T17:09:50.292043+00:00', '--job-id', '544', '--raw', '--subdir', 'DAGS_FOLDER/fire_incidents_dag.py', '--cfg-path', '/tmp/tmpszgz05_u']
[2025-03-09T17:10:04.369+0000] {standard_task_runner.py:105} INFO - Job 544: Subtask load_data
[2025-03-09T17:10:04.421+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_fire_incidents_dag.load_data manual__2025-03-09T17:09:50.292043+00:00 [running]> on host fad361e322ea
[2025-03-09T17:10:04.504+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_fire_incidents_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T17:09:50.292043+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T17:09:50.292043+00:00'
[2025-03-09T17:10:04.506+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-09T17:10:04.506+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-09T17:10:04.506+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2025-03-09 17:10:04.332926+00:00
[2025-03-09T17:10:04.507+0000] {logging_mixin.py:190} INFO - Dag name:etl_fire_incidents_dag and current dag run status:running
[2025-03-09T17:10:04.507+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-09T17:10:04.517+0000] {logging_mixin.py:190} INFO - Loading NYC Fire Incidents Data to Postgres DB....
[2025-03-09T17:10:04.518+0000] {warnings.py:112} WARNING - /opt/***/dags/pull_fire_incidents.py:190: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(load_data)

[2025-03-09T17:10:04.546+0000] {logging_mixin.py:190} INFO - Let us see the dates!!!
[2025-03-09T17:10:04.548+0000] {logging_mixin.py:190} INFO - 0    2021-01-04T00:01:00.000
1    2021-01-04T00:01:00.000
2    2021-01-04T00:01:00.000
Name: incident_datetime, dtype: object
[2025-03-09T17:10:04.576+0000] {logging_mixin.py:190} INFO - 
CREATE TABLE fire_incidents_schema (
	starfire_incident_id BIGINT, 
	incident_datetime TEXT, 
	alarm_box_borough TEXT, 
	alarm_box_number BIGINT, 
	alarm_box_location TEXT, 
	incident_borough TEXT, 
	zipcode BIGINT, 
	policeprecinct BIGINT, 
	citycouncildistrict BIGINT, 
	communitydistrict BIGINT, 
	communityschooldistrict BIGINT, 
	congressionaldistrict BIGINT, 
	alarm_source_description_tx TEXT, 
	alarm_level_index_description TEXT, 
	highest_alarm_level TEXT, 
	incident_classification TEXT, 
	incident_classification_group TEXT, 
	dispatch_response_seconds_qy BIGINT, 
	first_assignment_datetime TEXT, 
	first_activation_datetime TEXT, 
	incident_close_datetime TEXT, 
	valid_dispatch_rspns_time_indc TEXT, 
	valid_incident_rspns_time_indc TEXT, 
	incident_response_seconds_qy BIGINT, 
	incident_travel_tm_seconds_qy BIGINT, 
	engines_assigned_quantity BIGINT, 
	ladders_assigned_quantity BIGINT, 
	other_units_assigned_quantity BIGINT, 
	first_on_scene_datetime TEXT
)
[2025-03-09T17:10:04.784+0000] {logging_mixin.py:190} INFO - Batch Number 1 Loaded to Postgres.....
[2025-03-09T17:10:04.784+0000] {logging_mixin.py:190} INFO - Load is Complete
[2025-03-09T17:10:04.785+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-09T17:10:04.793+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-09T17:10:04.794+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_fire_incidents_dag, task_id=load_data, run_id=manual__2025-03-09T17:09:50.292043+00:00, execution_date=20250309T170950, start_date=20250309T171004, end_date=20250309T171004
[2025-03-09T17:10:04.816+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-09T17:10:04.817+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-09T17:10:04.817+0000] {logging_mixin.py:190} INFO - Dag name:etl_fire_incidents_dag queued_at:2025-03-09 17:09:50.316371+00:00
[2025-03-09T17:10:04.817+0000] {logging_mixin.py:190} INFO - Task hostname:fad361e322ea operator:PythonOperator
[2025-03-09T17:10:04.832+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-09T17:10:04.845+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-09T17:10:04.849+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
